tests/test_plots.py

```diff
@@ -3,6 +3,7 @@
 import matplotlib.pyplot as plt
 import numpy as np
 import matplotlib
+import ioos_qc.qartod as ioos_qc
```

Comment:
this is a very confusing way to do an import. How about `from ioos_qc import qartod` ?